index.html

```diff
@@ -0,0 +1,178 @@
+          Main Program
+        </h3>
+        <h3 class="baseline text-center text-white fw-bold"></h3>
+        <div class="align-items-center d-flex flex-column mt-5">
+          <div class="d-flex flex-column flex-md-row">
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/lecture.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">
+                Graphic Design Course
+              </p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/workshop.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">UI/UX Design Tools</p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+            <div
+              class="block blocke my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/forum.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">Forum to Discuss</p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/ignite.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">HTML/CSS Editors</p>
+              <p class="program-description text-white">
+                Tools for editing HTML/CSS code, making web design easier.
+                Examples: Sublime Text, Atom, Brackets, Dreamweaver, Notepad++.
+              </p>
+            </div>
+          </div>
+
+          <button class="join p-4 mt-5">Join WebDev Bootcamp 2023</button>
+          <p class="desktop-buttone">
+            <a href="" class="desktop-button">SEE THE WHOLE PROGRAM</a>
+          </p>
+        </div>
+      </article>
+      <artical class="speakers mt-5 px-4 py-5 bg-white">
+        <h3 class="main-program text-center text-black">Featured Speakers</h3>
+        <h3 class="baseline text-center text-white fw-bold"></h3>
```

Comment:
- [ ] As I suggested earlier,according to the semantic HTML rules, kindly make sure to use h3 only if you already have h2 :recycle:

Reply:
Done